src/Dialect/Mlir/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "s390x")
+  # currently needed for a bug on Z for the ONNXOps.cpp.inc file
```

Comment:
Since this file is not here, does that mean you don't need the block at all?

Reply:
yes, I removed it and the CI will confirm that if it is ok to remove it.